### DISCUSSION TITLE
msg="exec: \"docker\": executable file not found in %PATH%"

### DISCUSSION THREAD

**Discussion by jaffe-fly** [JetBrains/Qodana#97] [Q&A] [answered]
use  Qodana in pycharm terminal

![image](https://user-images.githubusercontent.com/49515380/176118687-72747ed5-f3c5-4a61-a597-30a9673c3c4c.png)

Documentation is not detailed enough for in windows platform

download 'qodana_windows_x86_64' file  and add the path in env path ，Do I have any other steps to take?

**Comment by brichbash** [comment, marked as answer]
Hello, 

Qodana requires Docker to run. You can get it at https://www.docker.com/get-started/
